### PR TITLE
Updated screenshots and fix auth

### PR DESCRIPTION
**Pull request by theckang**
Fixed openshift welcome screenshot (the user view, not admin view), updated kiali views, and added profile v2 image so workshop users know what the UI change should be for v2.

### CODE REVIEW

**Comment by theckang** [comment]
Redirect uri fixed by keeping `http://` prepend

**Comment by theckang** [comment]
Use alternate `reencrypt` kiali route so istio accepts verified cert.

File: content/workshops/openshift_service_mesh/lab5.3_prep.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ keycloak-0                                 1/1     Running           0
 </blockquote>
 
 ```
-sed "s|http://istio-ingressgateway-istio-system.apps.cluster.domain.com|$GATEWAY_URL|" ./istio-configuration/sso-realm.yaml | oc create -f -
```
Comment (dudash) [inline content/workshops/openshift_service_mesh/lab5.3_prep.md:58]: Does your GATEWAY_URL not include "http://"? - I merged but probably should've asked first. For me this sed seems to be correct.

Comment (theckang) [reply content/workshops/openshift_service_mesh/lab5.3_prep.md:58]: ugh sorry yes, you are right.  we need to revert it.

**Pull request merged**